README.md

```diff
@@ -52,20 +51,18 @@ firewall_ports:
 ...
 ```
 
-
 ## Elasticsearch/Kibana Upstream Repositories
 
-This playbook will add an upstream repository for
-Elasticsearch and Kibana unless you set variables to prevent it:
-
-As stated in <cite>[Ansible-Elasticsearch][1]</cite>.
-> "`es_use_repository` Setting this to `false` will stop Ansible from using the official Elastic package from any repository configured on the system
+This playbook adds an upstream repository for Elasticsearch and Kibana unless you set variables to prevent it:
 
-> `es_add_repository` Setting this to `false` will stop Ansible from adding the official Elastic package repositories (if 
-> es_use_repository is true) if you want to use a repo already present"
+As stated in [github.com/elastic/ansible-elasticsearch](https://github.com/elastic/ansible-elasticsearch):
+> "`es_use_repository`: Setting this to `false` will stop Ansible from using the official Elastic package from any repository configured on the system.
 
+> `es_add_repository`: Setting this to `false` will stop Ansible from adding the official Elastic package repositories (if `es_use_repository` is `true`) if you want to use a repository already present".
 
-`es_add_repository` set via `group_vars/all.yml` will also influence the existence of the Elasticsearch/Kibana repository on Kibana hosts. The default value inside the `kibana` role is `true`.
+> `es_add_repository`: Set via `group_vars/all/main.yml` will also influence the existence of the Elasticsearch/Kibana repository on Kibana hosts. The default value inside the `kibana` role is `true`.
```

Comment:
This is not a direct quote from another repo as the two points above are, so I would not use Markdown Quote syntax here

Reply:
Oh, my bad. Will fix and rebase.